deploy/helm/sumologic/README.md

```diff
@@ -482,5 +482,25 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `debug.metrics.collector.print`                                                       | Set to `true` to print logs processed by metric collector on standard output.                                                                                                                                                                                                                              | `false`                                                                                                                                                                              |
+| `debug.metrics.collector.stopLogsIngestion`                                           | Set to `true` to prevents sending metric collector logs to Sumo Logic. It prevents incresing ingestion and costs in case of huge amount of metric collector logs.                                                                                                                                          | `false`                                                                                                                                                                              |
+| `debug.metrics.metadata.forwardToSumologicMock`                                       | Set to `true` to forward metrics processed by metadata statefulset also to Sumo Logic Mock.                                                                                                                                                                                                                | `false`                                                                                                                                                                              |
+| `debug.metrics.metadata.print`                                                        | Set to `true` to print logs processed by metric metadata pods on standard output.                                                                                                                                                                                                                          | `false`                                                                                                                                                                              |
+| `debug.metrics.metadata.stopLogsIngestion`                                            | Set to `true` to prevents sending metric metadata logs to Sumo Logic. It prevents incresing ingestion and costs in case of huge amount of metric metadata logs.                                                                                                                                            | `false`                                                                                                                                                                              |
```

Comment:
```suggestion
| `debug.metrics.metadata.stopLogsIngestion`                                            | Set to `true` to prevent sending metric metadata logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of metric metadata logs.                                                                                                                                            | `false`                                                                                                                                                                              |
```